src/cursor/cur_backup.c

```diff
@@ -82,14 +82,20 @@ __curbackup_close(WT_CURSOR *cursor)
 
 	CURSOR_API_CALL(cursor, session, close, NULL);
 
-	WT_TRET(__backup_cleanup_handles(session, cb));
+	/*
+	 * If we got far enough to hold the hot backup lock, then we own all
+	 * of the necessary cleanup (on error or success).
+	 */
```

Comment:
I'd like a sentence here or where we set the `WT_CURBACKUP_LOCKER` flag that explicitly states that it will never be cleared.  It gets "cleared" implicitly by calling `wt_cursor_close` which frees the cursor.